packages/bot-web-ui/src/components/journal/journal-components/__tests__/journal-item.spec.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { message_types, log_types } from '@deriv/bot-skeleton';
+import JournalItem from '../journal-item';
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({}));
+
+/* eslint-disable testing-library/no-node-access */
+jest.mock('react-transition-group', () => {
+    const FakeTransition = jest.fn(({ children }) => children);
+    const FakeCSSTransition = jest.fn(props =>
+        props.in ? (
+            <FakeTransition>
+                <div data-testid='mock-css-transition'>{props.children}</div>
+            </FakeTransition>
+        ) : null
+    );
+    return { CSSTransition: FakeCSSTransition, Transition: FakeTransition };
+});
```

Comment:
this is taken from an example of official documentation how to mock `CSSTransition` from 'react-transition-group'. You can take a look here:
https://testing-library.com/docs/example-react-transition-group/

Reply:
@maryia-matskevich-deriv just for understanding, why do we need to mock `react-transition-group` without mocking it is there any issue that we are facing here?

Reply:
@shafin-deriv Myself and @vinu just tried cleaning it up and it too works and covers 100%, some kind of mysticism.😳 component didn't want to render in any way without this moc a few days ago, I and guys tried different paths...